configuration/redis/redis.go

```diff
@@ -242,6 +243,10 @@ func (r *ConfigurationStore) Get(ctx context.Context, req *configuration.GetRequ
 
 		redisValue, err := r.client.Get(ctx, redisKey).Result()
 		if err != nil {
+			if strings.Contains(err.Error(), redisWrongTypeIdentifyStr) {
```

Comment:
@LaurenceLiZhixin should we add a unit test for this scenario?

Reply:
Yes I added it at https://github.com/dapr/components-contrib/pull/1693/files#diff-8d2a56ddd092485603a1d00e47936c985c2baebae58884df2f0e51ea408d75b9R263
User can have any type of value in redis when using configuraiton api.
@shubham1172

Reply:
Should it be a part of `setupMiniRedis`? Or should we add a new test scenario: `get does not throw error for wrong type`?

Reply:
OK, I would fix it.

Reply:
Thanks!